services/settings/pkg/config/config.go

```diff
@@ -45,4 +46,16 @@ type Metadata struct {
 	SystemUserID     string `yaml:"system_user_id" env:"OCIS_SYSTEM_USER_ID;SETTINGS_SYSTEM_USER_ID" desc:"ID of the oCIS STORAGE-SYSTEM system user. Admins need to set the ID for the STORAGE-SYSTEM system user in this config option which is then used to reference the user. Any reasonable long string is possible, preferably this would be an UUIDv4 format."`
 	SystemUserIDP    string `yaml:"system_user_idp" env:"OCIS_SYSTEM_USER_IDP;SETTINGS_SYSTEM_USER_IDP" desc:"IDP of the oCIS STORAGE-SYSTEM system user."`
 	SystemUserAPIKey string `yaml:"system_user_api_key" env:"OCIS_SYSTEM_USER_API_KEY" desc:"API key for the STORAGE-SYSTEM system user."`
+	Cache            *Cache `yaml:"cache"`
+}
+
+// Cache configures the cache of the Metadata store
+type Cache struct {
+	Store          string        `yaml:"store" env:"OCIS_CACHE_STORE;SETTINGS_CACHE_STORE" desc:"The type of the cache store. Supported values are: 'memory', 'ocmem', 'etcd', 'redis', 'redis-sentinel', 'nats-js', 'noop'. See the text description for details."`
+	Nodes          []string      `yaml:"addresses" env:"OCIS_CACHE_STORE_NODES;SETTINGS_CACHE_STORE_NODES" desc:"A comma separated list of nodes to access the configured store. This has no effect when 'memory' or 'ocmem' stores are configured. Note that the behaviour how nodes are used is dependent on the library of the configured store."`
+	Database       string        `yaml:"database" env:"OCIS_CACHE_DATABASE" desc:"The database name the configured store should use."`
```

Comment:
```suggestion
	Database       string        `yaml:"database" env:"SETTINGS_CACHE_DATABASE" desc:"The database name the configured store should use."`
```
afaik there is no global var for that

Reply:
@kobergj  I was also wondering. But `frontend`, `gateway`, `proxy`, `storage-system` and `storage-users` all use `OCIS_CACHE_DATABASE` (without any local override).

Reply:
Funny. `userlog`, `eventhistory`, `graph` and `postprocessing` all use local Envvars without global override :smile:

Reply:
Hm, those are `STORE_DATABASE` settings as oposed to `CACHE_DATABASE`. Well except `GRAPH_CACHE_STORE_DATABASE`, which .... :thinking:  :exploding_head:

Reply:
Fine for me to keep as is for now